pkg/printers/internalversion/printers.go

```diff
@@ -1193,14 +1205,19 @@ func getNodeExternalIP(node *api.Node) string {
 
 // findNodeRole returns the role of a given node, or "" if none found.
 // The role is determined by looking in order for:
+// * a node-role.kubernetes.io/master label
 // * a kubernetes.io/role label
 // * a kubeadm.alpha.kubernetes.io/role label
-// If no role is found, ("", nil) is returned
 func findNodeRole(node *api.Node) string {
-	if role := node.Labels[kubeadm.NodeLabelKubeadmAlphaRole]; role != "" {
-		return role
+	if _, ok := node.Labels[LabelNodeRoleMaster]; ok {
+		return "Master"
+	}
+	if role := node.Labels[NodeLabelRole]; role != "" {
+		return strings.Title(role)
+	}
+	if role := node.Labels[NodeLabelKubeadmAlphaRole]; role != "" {
+		return strings.Title(role)
```

Comment:
@alexandercampbell To make first letter of words uppercase, such as "node" -> "Node".